### PR TITLE
Split inferentia CI workflows

### PR DESCRIPTION
**Pull request by dacorvo**
These workflows take too long: they need to be split.

@JingyaHuang for some reason, one of the stable diffusion test is failing repeatedly, while it was not before ...

### CODE REVIEW

File: .github/workflows/test_inf2.yml [hunk 1]
```diff
@@ -47,15 +47,3 @@ jobs:
         run: |
           source aws_neuron_venv_pytorch/bin/activate
           HF_TOKEN=${{ secrets.HF_TOKEN_OPTIMUM_NEURON_CI }} pytest -m is_inferentia_test tests/generation
-      - name: Run exporters tests
```
Comment (JingyaHuang) [inline .github/workflows/test_inf2.yml:50]: could we name the file `test_inf2_common` and job name with `name: Optimum neuron / Test INF2 Common`?

**Review by JingyaHuang — COMMENTED** [review]
LGTM, just left one nit on the naming, thanks for the PR!

File: .github/workflows/test_inf2.yml (hunk 1 above)
Comment (dacorvo) [reply .github/workflows/test_inf2.yml:50]: I agree test_inf2.yml is a poor name, but these are not really common tests: all decoder tests are included. I was about to call it test_inf2_decoder, but there are some CLI tests also, so I kept the name ...

Comment (JingyaHuang) [reply .github/workflows/test_inf2.yml:50]: Oki doki 😤, then we keep it this way for now.

**Review by JingyaHuang — APPROVED** [review]

**Pull request closed without merging**